src/react-app-env.d.ts

```diff
@@ -1,5 +1,72 @@
+declare namespace NodeJS {
+  interface ProcessEnv {
+    readonly NODE_ENV: 'development' | 'production' | 'test';
+    readonly PUBLIC_URL: string;
```

Comment:
Are we keeping this or not?

Reply:
no, not needed, and I've moved this whole file to typings/webpack.d.ts, and removed almost everything from it